base/config-info.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: dashboard-info
+  namespace: tekton-pipelines
+  labels:
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/part-of: tekton-dashboard
+data:
+  # Contains dashboard version which can be queried by external
+  # tools such as CLI. Elevated permissions are already given to
+  # this ConfigMap such that even if we don't have access to
+  # other resources in the namespace we still can have access to
+  # this ConfigMap.
+  version: "devel"
```

Comment:
We'll need to update this in the release pipeline so "devel" is replaced with the correct version label, see https://github.com/tektoncd/dashboard/blob/97d983f043ea7a1f23d88d408c992f9f981bc789/tekton/publish.yaml#L79-L81

@nikhil-thomas fyi, since you're working on the release pipeline at the moment. It's a small change so hopefully there'll be no conflicts.

Reply:
Did the changes @AlanGreene, thanks for your review and feedback